lib/resources/r4/observation.yaml

```diff
@@ -358,6 +387,7 @@ fields:
   note: |
     <ul>
       <li> Must be a reference to Practitioner. </li>
+      <li> Vital Signs require performer. </li>
```

Comment:
We might be able to leave this out. There's a change coming to remove this restriction

Reply:
https://github.com/cerner/fhir.cerner.com/pull/675/commits/52e09da54bd018cba858d003d63e103e1bf4a2a9#diff-e303aa1b2a83c6a68a12604d4956dcd3c31f4ffe40546bbcc31e2cc6588921dcL390